src/components/AcceptOrRejectAllOrMoreChoices.tsx

```diff
@@ -68,14 +73,21 @@ export function AcceptOrRejectAllOrMoreChoices({
           </p>
         </div>
       </div>
-      <div className="accept-or-reject-all-button-row">
+      <div className="accept-or-reject-all-button-row" role="none">
         <Button
           primaryText={formatMessage(messages.acceptAllButtonPrimary)}
           handleClick={handleAcceptAll}
+          ariaDescription={formatMessage(
+            messages.acceptAllButtonAriaDescription,
+          )}
         />
         <Button
           primaryText={formatMessage(messages.rejectAllButtonPrimary)}
           handleClick={handleRejectAll}
+          ariaDescription={formatMessage(
+            messages.rejectAllButtonAriaDescription,
+          )}
+          autoFocus
         />
         <Button
```

Comment:
it doesnt quite apply since it completely replaces the entire contents of the dialog, its not an expanded/hidden dichotomy. if you have a better role to tag that with lmk, i couldnt really nail anything down

Reply:
Ah, I see. You're probably right about that. You might see if the [tab interface](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tab_role) applies, but my guess is not and I don't have a better suggestion. 🤷‍♀️

Reply:
yeah, not quite.... maybe an aria-controls that points to the dialog element?? idk

Reply:
Yeah, I was going to suggest something like that, but if the button isn't a direct toggle (e.g. a different button shows the content than the one that hides it) and/or if the thing that appears/disappears isn't the same div then idk if it applies.

Reply:
Also, it's very hard for me to say since I'm not familiar with the product. Happy to trust whatever you researched and considered here.

Reply:
imo, just from a common sense standpoint, if youre hitting a button that says "more choices" youre probably aware that youre going to be receiving some sort of context change? adding some sort of half-correct aria tag might just do more harm than good here

Reply:
Yep, totally agree.